test/ng/documentSpec.js

```diff
@@ -27,3 +27,29 @@ describe('$document', function() {
     });
   });
 });
+
+
+describe('$$isDocumentHidden', function() {
```

Comment:
I agree, but I found it quite complicated to do that. Because you'd have to mock the document (because hidden isn't writable), and still have it accept event listeners and fire events.